eng/common/Update-Change-Log.ps1

```diff
@@ -12,7 +12,8 @@ param (
   [Parameter(Mandatory = $true)]
   [String]$ChangeLogPath,
   [String]$Unreleased = $True,
-  [String]$ReplaceVersion = $False
+  [String]$ReplaceVersion = $False,
+  [String]$ReleaseDate
```

Comment:
@sima-zhu this change while not part of your new tools PR showed up because your other sync PR https://github.com/Azure/azure-sdk-for-python/pull/13368/files accidently reverted this file when you merged it.

Reply:
FYI @chidozieononiwu

Reply:
Will take extra look next time reviewing the changes